sphinx/builders/html.py

```diff
@@ -1403,7 +1403,7 @@ def setup(app):
     app.add_config_value('html_favicon', None, 'html', [str])
     app.add_config_value('html_css_files', [], 'html')
     app.add_config_value('html_js_files', [], 'html')
-    app.add_config_value('html_static_path', [], 'html')
+    app.add_config_value('html_static_path', ['_static'], 'html')
```

Comment:
I modified the default value of `html_static_path` to remove the definition from `conf.py`.

Reply:
OMG, this settings force to users to create `_static` directory on Sphinx project even if they does not use quickstart...
That must not be good. I'll revert this change ASAP.